### PR TITLE
libvirt: 6.0.0 -> 6.1.0, fix module

### PR DESCRIPTION
**Pull request by volth**
`libvirtd` rejects to start with `--listen` flag in presence of systemd, so I switched to socket activation.
it works on my laptop, but I have not tested yet various cases like restoring bridges after nixos-rebuild

cc @Izorkin

### CODE REVIEW

File: nixos/modules/virtualisation/libvirtd.nix [hunk 1]
```diff
@@ -214,14 +214,14 @@ in {
     };
 
     systemd.services.libvirtd = {
-      description = "Libvirt Virtual Machine Management Daemon";
-
-      wantedBy = [ "multi-user.target" ];
       requires = [ "libvirtd-config.service" ];
       after = [ "systemd-udev-settle.service" "libvirtd-config.service" ]
-              ++ optional vswitch.enable "ovs-vswitchd.service";
+              ++ optional vswitch.enable "vswitchd.service";
```
Comment (volth) [inline nixos/modules/virtualisation/libvirtd.nix]: oops,

**Comment by Izorkin** [comment]
@volth freezing start OS.
```[   ***] A start job is running for Bridge Interface br0 (1min 28s / no limit)```

**Comment by volth** [comment]
> @volth freezing start OS.
> `[ ***] A start job is running for Bridge Interface br0 (1min 28s / no limit)`

and now?

File: nixos/modules/tasks/network-interfaces-scripted.nix [hunk 2]
```diff
@@ -291,13 +291,19 @@ let
...
-                  for uri in qemu:///system lxc:///; do
-                    for dom in $(${pkgs.libvirt}/bin/virsh -c $uri list --name); do
-                      ${pkgs.libvirt}/bin/virsh -c $uri dumpxml "$dom" | \
-                      ${pkgs.xmlstarlet}/bin/xmlstarlet sel -t -m "//domain/devices/interface[@type='bridge'][source/@bridge='${n}'][target/@dev]" -v "concat('ip link set ',target/@dev,' master ',source/@bridge,';')" | \
-                      ${pkgs.bash}/bin/bash
+                  #
+                  # if `libvirtd.service` is not running, do not use `virsh` which would try activate it via 'libvirtd.socket' and thus start it out-of-order.
+                  # `libvirtd.service` will set up bridge interfaces when it will start normally.
+                  #
+                  if ${pkgs.systemd}/bin/systemctl status 'libvirtd.service'; then
```
Comment (volth) [inline nixos/modules/tasks/network-interfaces-scripted.nix]: `systemctl is-active libvirtd.service` would be better, but is ok for now just to check the idea

**Comment by Izorkin** [comment]
Thanks! This variant worked.

**Comment by wedens** [comment]
Thanks for helping to move this forward!

**Pull request merged**